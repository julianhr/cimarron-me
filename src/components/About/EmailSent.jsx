import React from 'react'
import PropTypes from 'prop-types'
import styled from '@emotion/styled'

import imgSrc from '~/assets/img/thumbs_up.png'
import ImageLoader from '~/library/ImageLoader'


const Root = styled.div`
  align-self: center;
  display: flex;
  flex-direction: column;
  align-items: center;
  padding: 80px 0 0;

  h3 {
    color: ${props => props.theme.colors.text};
  }
`


function EmailSent() {
  return (
    <Root>
      <ImageLoader
        maxWidth={339}
        maxHeight={364}
        imgSrc={imgSrc}
        shouldFadeIn={false}
      />
      <h3
        css={{ marginBottom: 0 }}
      >
        Message successfully sent
      </h3>
      <h3>Thank you!</h3>
    </Root>
  )
}

export default EmailSent
