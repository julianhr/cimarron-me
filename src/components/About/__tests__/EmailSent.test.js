import React from 'react'
import { render, mount, shallow } from 'enzyme'
import { create } from 'react-test-renderer'

import EmailSent from '../EmailSent'
import MockApp from '~/__tests__/__mocks__/MockApp'


describe('EmailSent', () => {
  it('matches snapshot', () => {
    const wrapper = render(
      <MockApp>
        <EmailSent />
      </MockApp>
    )

    expect(wrapper).toMatchSnapshot()
  })
})
