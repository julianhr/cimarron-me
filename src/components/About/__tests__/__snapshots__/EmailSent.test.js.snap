// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmailSent matches snapshot 1`] = `
<div
  class="css-9okclz-Root e11ghwax0"
>
  <figure
    class="css-yxmayv-Root-ImageLoader eeswxza0"
  >
    <img
      class="css-0"
      src="test-file-stub"
    />
  </figure>
  <h3
    class="css-tecitd-EmailSent"
  >
    Message successfully sent
  </h3>
  <h3>
    Thank you!
  </h3>
</div>
`;
